Persistence/Migrations/20230626190616_AddFinancialYearAsAggregateRoot.cs

```diff
@@ -0,0 +1,92 @@
+﻿using System;
+using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Persistence.Migrations
+{
+    /// <inheritdoc />
+    public partial class AddFinancialYearAsAggregateRoot : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.AddColumn<Guid>(
+                name: "FinancialYearId",
+                schema: "HaSpMan",
+                table: "Transactions",
+                type: "uniqueidentifier",
+                nullable: true);
```

Comment:
This shouldn't be nullable eventually, but of course it must be temporarily nullable for all existing transactions. However the should be some migration SQL code in here to add the column (with nullable: true), update the value for all existing transactions, then make it non-nullable for all new transactions